tests/helpers/unittest_base.py

```diff
@@ -13,11 +15,21 @@
 )
 from tests.helpers.mysql_helper import reset_mysql_instance
 
+
 class BaseTestCase(unittest.TestCase):
     """ Base class for unit tests."""
 
-    def setUp(self):
+    def __init__(self, *args, **kwargs):
+        super(BaseTestCase, self).__init__(*args, **kwargs)
+        # make sure this gets called before anything else.
         reset_mysql_instance()
+        self.fake = faker.Faker()
+
+    def setUp(self) -> None:
+        super(BaseTestCase, self).setUp()
+
+        reset_mysql_instance()
```

Comment:
I don't think we want this in __init__ and setup do we ?

Reply:
Part of the problem on Friday was the `setUp()` method test_site_dao.py didn't call `reset_mysql_instance()`. We don't really need to call `reset_mysql_instance()` in the init method, we just need to set the `UNITTEST_FLAG` environment variable in the init method.  I've added a new commit that fixes that.